.github/workflows/test.yml

```diff
@@ -0,0 +1,20 @@
+name: Tests
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    name: Unit Tests
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.13.1'
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - run: git remote add upstream https://github.com/elastic/ecs
```

Comment:
> The remote addition is so that CI will actually work on forks that don't have the same tags as the elastic/ecs repo, you might be able to remove it.

Yes, I ran into the same problem with my fork while testing.

Do you see a need for testing to work on forks? I'm leaning towards removing it; all the tests can be ran locally and rely on CI once opening a PR.

Reply:
Not particularly, I just can never recall how the branch references work in terms of PRs -- if the `git fetch --prune --unshallow --tags` gets run in the context of the repo of the PR author rather than in the context of the `elastic` org with changes overlayed on it, then it's not going to work if the user doesn't have the latest build tags.

Want me to remove it and then add it back in if things don't work?

Reply:
Yeah, let's remove it. We can easily re-add if we run into issues.